assets/helpers/logger.js

```diff
@@ -5,7 +5,7 @@ import Raven from 'raven-js';
 
 export const init = () => {
   const dsn: string = 'https://dc13eb8698614a8081ce6a139d9f4aab@sentry.io/171710';
-  const gitCommitId = window.guardian.gitCommitId;
+  const { gitCommitId } = window.guardian;
```

Comment:
Wait, what? Is `window.guardian` only the `gitCommitId`?

Reply:
The `gitCommitId` is in the `window.guardian` object.

Reply:
The new set of rules enforce to use [destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment) whenever is possible. If we as a team don't like it we can turn it off in the future. Since this PR is quite big, I would like to merge with this property on, discuss the best approach and then turn it off or keep it on in a follow up PR.